src/test/scala/com/fulcrumgenomics/util/SequencesTest.scala

```diff
@@ -48,25 +49,46 @@ class SequencesTest extends UnitSpec {
     Sequences.countMismatches("ACGACTATATGCAT", "NNNNNNNNNNNNNN") shouldBe 14
   }
 
-  "Sequences.longestHomopolymer" should "fail when passed a null String" in {
-    an[Throwable] shouldBe thrownBy { Sequences.longestHomopolymer(null) }
+  "Sequences.longestHomopolymer" should "fail when passed a zero length string" in {
+    an[Throwable] shouldBe thrownBy { Sequences.longestHomopolymer("") }
   }
 
-  it should "fail when passed a zero length string" in {
-    an[Throwable] shouldBe thrownBy { Sequences.longestHomopolymer("") }
+  "Sequences.maxPolyX" should "fail when passed a null String" in {
+    an[Throwable] shouldBe thrownBy { Sequences.maxPolyX(null) }
+  }
+
+  it should "return zero when passed a zero length string" in {
+    Sequences.maxPolyX("") shouldBe OffsetAndLength(0, 0)
```

Comment:
Why would you want this behavior?  Also, I don't think it's reasonable to deprecate one method in favor of another if the behavior is different.

Reply:
1. Sometimes I have case classes with an empty sequence, so I don't want it to blow up when calling `maxPolyX`: 
```
things.map(maxPolyX(_.sequence))
```
The longest homopolymer in an empty sequence is the empty string isn't it?
2. I don't think you should be using this method to catch the case (i.e. throw an exception) when you have an empty string.  I think it's wrong, so I'd like to fix it. I really don't want to have to write guards everywhere in my code to ignore feeding empty strings to `longestHomopolymer`.  One could argue that asking for the longest homopolymer implies that there is at least one homopolymer, but I think that's just an argument for argument's sake.  

Any objections to fixing this?   Do you have code that will break (i.e. relies on the exception being thrown here)?